package.json

```diff
@@ -23,6 +26,8 @@
     "https-proxy-agent": "^2.2.2",
     "jest-transform-graphql": "^2.1.0",
     "lodash-es": "^4.17.21",
+    "postcss": "^7",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat",
```

Comment:
Just wondering why the format of this is different to all the other dependencies?

Reply:
There is an earlier dependency in one of the existing packages in Traction that relies on an older version of PostCSS. So we need to use postcss7-compat along with tailwind, that is intended to use for this purpose.